doc/topics/releases/3000.rst

```diff
@@ -33,6 +33,12 @@ we continue to work on upgrading Tornado to a recent version. New code that
 uses Tornado should import the module from ``salt.ext.tornado`` instead of
 importing the system version of Tornado.
 
+Msgpack 1.0.0rc1 Incompatibility
+================================
+Salt is currently incompatible with mgspack 1.0.0rc1. We recommend using
+versions <= 1.0.0rc1. This impacts the Salt Master's ability to run commands
```

Comment:
versions < 1.0

Reply:
> versions < 1.0

this

Reply:
thanks for catching that. updated. i put 1.0.0 instead of 1.0 as that looks to be the versioning scheme

Reply:
:+1: